dl/src/main/scala/com/intel/analytics/bigdl/models/rnn/Train.scala

```diff
@@ -85,17 +85,18 @@ object Train {
       }
 
       Engine.init(1, param.coreNumber, false)
+      val criterion = new CrossEntropyCriterion[Float](squeezeFlag = true)
```

Comment:
I don't think we should change the Loss interface at this time. Ideally the optimizer should provide the workingCriterion(i) for the validation method to use; we can leave that to future improvement.